app/models/setting/general.rb

```diff
@@ -28,4 +28,10 @@ def self.load_defaults
 
     true
   end
+
+  def self.validate_settings(errors)
+    ["idle_timeout", "proxy_request_timeout", "entries_per_page"].each do |s|
```

Comment:
This is unrelated, should be in its own PR... but it also should be using the regular model validations.  This validate_settings thing will be confusing if we use it for both validation of multiple inter-dependent settings and validation of standalone settings, which we already do today in models.